experimental/conf.py

```diff
@@ -135,10 +138,28 @@
 # POPULATE LINKS TO OTHER VERSIONS
 html_context['versions'] = list()
 
-tags = sorted(repo.tags, key=lambda t: t.commit.committed_datetime)
-versions = [branch.name for branch in tags]
-versions = versions[len(versions)-1:]
+tags = sorted(repo.tags, key=lambda t: t.commit.committed_datetime, reverse=True)
+versions = list()
+currentMinor = -1
+count = 0
+for tag in tags:
+    current = Version(tag.name)
+    if current.minor != currentMinor and current.minor >= 8:
+        versions.append(tag.name)
+        currentMinor = current.minor
+        count = count + 1
+        if count >= 2:
+            break
```

Comment:
How about the following, just to make it a bit more portable?

```suggestion
def comparator(version: str):
    """
    This comparator compares only major and minor versions of a SemVer string.
    """
    version = Version(version)
    return (version.major, version.minor)

def keep_latest_n_versions(versions: List[str], keep: int = 2):
    """
    Return the latest semantic version from each major -> minor group.
    Note that the versions don't need to be sorted.
    """
    versions = sorted(T, key=lambda v: comparator(v), reverse=True)
    got, res = 0, []
    for _, group in groupby(versions, lambda x: (Version(x).major, Version(x).minor)):
        if got >= keep:
            return res
        res.append(list(group)[0])
        got += 1
    return res

tag_names = [Version(tag.name) for tag in tags]
versions = keep_latest_n_versions(tag_names)
```